pyproject.toml

```diff
@@ -11,8 +11,10 @@ python = "^3.9"
 anytree = "^2.8.0"
 jax = "^0.4.4"
 jaxlib = "^0.4.4"
-numpy = "^1.24.2"
+numpy = "^1.23.0"
 typing-extensions = "^4.5.0"
+llvmlite = "^0.39.1"
+scphylo-tools = {git = "https://github.com/pawel-czyz/scphylo-tools.git"}
```

Comment:
When trying to install this branch I get some issues with installing these two dependencies. Any idea where the problem may arise from?

[Screenshot 2023-05-04 at 10 57 17](https://user-images.githubusercontent.com/10867778/236158109-3fff70be-2d49-4ba3-9a00-d83d8b9f3207.png)
[Screenshot 2023-05-04 at 10 57 33](https://user-images.githubusercontent.com/10867778/236158114-29ab7d4a-d1fe-4a6c-82c9-d74402a89b3c.png)

Reply:
Hmmm... perhaps installing `llvmlite` manually could help? Or if Poetry can't handle the dependency, you can try to install (in the Poetry environment, so `poetry run pip install llvmlite` or via `poetry shell`) it using plain `pip`

Reply:
managed to install `llvmlite` now in my poetry env now. - Not sure what worked in the end. 

I unfortunately I still can't run `poetry pytest` successfully, `scphylo`is still not recognized - despite my appempt of manually git clone and install it into poetry dirs directly.  Any suggestions?

Reply:
Do you happen to have original `scphylo` installed (can be visible as `scphylo-tools`)? Perhaps it already sees that there's a package like this and doesn't take the version from git.

Alternatively, you can try something like `poetry run pip install "scphylo @ git+https://github.com/pawel-czyz/scphylo-tools.git"` or `poetry run pip install "scphylo-tools @ git+https://github.com/pawel-czyz/scphylo-tools.git"`)